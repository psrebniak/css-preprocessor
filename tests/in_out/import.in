$item: 'first';
@import 'import_1.part';

$item: 'second';
@import 'import_1.part';

@import 'import_2.part';
