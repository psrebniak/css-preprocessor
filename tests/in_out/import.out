// Import file: 'import_1.part'
.test {
    test: 'first';
}
// Import file: 'import_1.part'
.test {
    test: 'second';
}
// Import file: 'import_2.part'
// Import file: 'import_1.part'
.test {
    test: 'third';
}
